miranda/hq/cf_attrs.json

```diff
@@ -0,0 +1,96 @@
+{
```

Comment:
I feel like having JSON files structuring the CF-like metadata like this for variables for other projects as well would make for a cleaner way of creating NetCDF files. What do you think?

Reply:
Possibly, time will tell. It takes a while to realize the shortcomings of a given solution.